### PR TITLE
Improve coverage reporting

### PR DESCRIPTION
**Pull request by marclove**
– Move spec run logic to script
– Add nyc to run mocha specs under istanbul

### CODE REVIEW

**Comment by marclove** [comment]
@camwest this should satisfy #24.

**Comment by camwest** [comment]
@marclove how can we write a test for the orion-specs script? It may seem annoying but all the other scripts have tests...

**Comment by marclove** [comment]
@camwest I think adding specs would be brittle and not very useful. I'd be stuck either asserting line-by-line my implementation or asserting stdout output, which I don't have full control over. If the spec script is broken, we're gonna know about it because the specs will fail.

File: packages/scripts/orion-specs.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+  } else {
+    console.log(`Non-component specs failed:
+      ${stdout && ','}
+      ${stderr}`);
+  }
+  console.log('================================================================================');
+}
+
+function mkBuildDir() {
+  const { code, stdout, stderr } = exec(`mkdir -p ${knownPaths.build}`);
```
Comment (marclove) [inline packages/scripts/orion-specs.js]: oh cool, thanks. will fix.

**Comment by camwest** [comment]
@marclove OK fair enough thanks

**Comment by camwest** [comment]
@marclove can you make this branch up-to-date with the base branch?

**Pull request merged**